com.unity.render-pipelines.universal/Editor/AssetPostProcessors/MaterialPostprocessor.cs

```diff
@@ -256,6 +256,23 @@ static void UpgradeV3(Material material, ShaderPathID shaderID)
 
         static void UpgradeV4(Material material, ShaderPathID shaderID)
         {}
+
+        static void UpgradeV5(Material material, ShaderPathID shaderID)
+        {
+            var propertyID = Shader.PropertyToID("_Surface");
+            if (material.HasProperty(propertyID))
+            {
+                float surfaceType = material.GetFloat(propertyID);
+                if (surfaceType >= 1.0f)
```

Comment:
Should this be an enum instead of 1? If it's enable/disable i.e. 1/0 case, then I'd probably test against 0.